### PR TITLE
add devops-engineer example

### PR DESCRIPTION
**Pull request by isbkch**
Another example for a DevOps engineer

Screenshot 2023-07-24 at 2 45 54 PM

### CODE REVIEW

File: examples/devops-engineer/Modelfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Modelfile for creating a tweet from a topic
+# Run `ollama create devops-engineer -f ./Modelfile` and then `ollama run devops-engineer` and enter a topic
+
+FROM llama2:13b
+PARAMETER temperature 1
+SYSTEM """
+
```
Comment (jmorganca) [inline examples/devops-engineer/Modelfile:7]:
```suggestion
```

**Comment by jmorganca** [comment]
Thanks for the PR! This looks super cool. You may want to remove the leading newline in the `SYSTEM` instruction – although let me know if that was by design and we can merge as-is.

File: examples/devops-engineer/Modelfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Modelfile for creating a tweet from a topic
```
Comment (mchiang0610) [inline examples/devops-engineer/Modelfile:1]: Might want to take out the comment from the creating a tweet example.

**Comment by mchiang0610** [comment]
This is super cool! Thank you for putting together an example

**Comment by isbkch** [comment]
Thank you for the comments/suggestions. They’re all implemented now

**Review by mchiang0610 — APPROVED** [review]
⛳️ Boom! Thank you.

**Pull request merged**